l2geth/core/state_transition.go

```diff
@@ -243,7 +269,13 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 		}
 	}
 	st.refundGas()
-	st.state.AddBalance(evm.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice))
+	if rcfg.UsingOVM {
```

Comment:
Leave a comment that `l2Fee` is the same as the normal fee that is charged in the non using ovm codepath